### PR TITLE
Add release process document

### PR DESCRIPTION
**Pull request by Cruikshanks**
This change adds a document that details the release process for the [charging-module-api](https://github.com/DEFRA/charging-module-api) and the [tactical-charging-module](https://github.com/DEFRA/sroc-tcm-admin).

It's primarily for use by the development team as it details actions they need to take. However, it should be of use to everyone on the team.

### CODE REVIEW

File: release_process.md [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+
+The development team will then review the changes made to decide whether a major, minor, or patch version bump is needed.
+
+> The 'version' agreed is what any reference to version below is referring to.
+
+### Update app version
+
+> IMPORTANT! Ensure you have checked out the main branch for the repo and performed a `git pull` to get the latest code first.
+
+We need to update the version held in the app to match our new version. This does mean changing a code file but you'll notice we don't create a PR. This and [updating the CHANGELOG](#update-the-changlog) are the exceptions.
```
Comment (StuAA78) [inline release_process.md:63]: Link is missing the `e` in `changelog`

**Review by StuAA78 — CHANGES_REQUESTED** [review]
Spotted a tiny typo in one of the links, all good other than that -- very thorough and clear! 😁

File: release_process.md (hunk 1 above)
Comment (Cruikshanks) [reply release_process.md]: 🤦

**Review by StuAA78 — APPROVED** [review]
👍

**Pull request merged**